packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -54,7 +61,10 @@ export async function createNewFluidFile(
     }
 
     const filePath = newFileInfo.filePath ? encodeURIComponent(`/${newFileInfo.filePath}`) : "";
-    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}.fluid`);
+    const containerPackageName = newFileInfo.containerPackageName;
```

Comment:
Is this newFileInfo from resolvedUrl or from the host?

Reply:
This is made from resolved url.